SummerProject/static/src/components/Sidebar/index.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import PropTypes from "prop-types";
+import axios from "axios";
```

Comment:
You'll need to import `axios` from `'../../axios'` and get rid of `http://localhost:8000` reference below (on line `13`) after Andrij merges his pull request - https://github.com/Liliia111/SummerSchool/pull/10/files#diff-545a7e790b07e9d282ffadbb6a6db045

Reply:
done